test/session_test.exs

```diff
@@ -38,9 +38,13 @@ defmodule Janus.SessionTest do
     }
   ]
 
+  setup :set_mox_global
+
   setup do
+    stub(DateTimeMock, :utc_now, &DateTime.utc_now/0)
+
     {:ok, connection} =
-      Connection.start(Janus.MockTransport, @request_response_pairs, FakeHandler, {})
+      Connection.start(Janus.MockTransport, @request_response_pairs, FakeHandler, [], [])
 
     %{connection: connection}
```

Comment:
setup on exit here
https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html#on_exit/2
It should work.

Reply:
Done